src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/advice/LogAndSuppressRequestRejectedExceptionFilter.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.lrdapi.controllers.advice;
+
+import com.microsoft.applicationinsights.core.dependencies.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.web.firewall.RequestRejectedException;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.GenericFilterBean;
+import uk.gov.hmcts.reform.lrdapi.controllers.constants.ErrorConstants;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@Component
+@Slf4j
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class LogAndSuppressRequestRejectedExceptionFilter extends GenericFilterBean {
+
+    private Gson gson = new Gson();
```

Comment:
can't we Autowire this?

Reply:
Not sure about @autowire.. this class copied from PRD and it's filter class not sure how to use autowire here?

Reply:
Removed the class which is not required here.